### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by robnester-rh**
Adding initial issue template.

### CODE REVIEW

**Review by lcarva — APPROVED** [review]
Cool!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+_What actually happened._
+
+## Screenshots or Terminal Output
+
+_If applicable, add screenshots or copy-paste the terminal output to help explain your problem._
+
+## Environment Details
+
+- **Operating System:**
+- **CLI Version:**
```
Comment (zregvart) [inline .github/ISSUE_TEMPLATE/bug_report.md:37]:
```suggestion
- **`ec` CLI Version, run `ec version --short`:**
```

**Review by zregvart — APPROVED** [review]
Hinting to how to specify the version would be helpful, LGTM otherwise...

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (robnester-rh) [reply .github/ISSUE_TEMPLATE/bug_report.md:37]: Great idea! Thanks!

**Pull request merged**